.readthedocs.yaml

```diff
@@ -0,0 +1,23 @@
+# .readthedocs.yaml
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+# Required
+version: 2
+
+# Build documentation in the docs/ directory with Sphinx
+sphinx:
+  configuration: docs/source/conf.py
+
+# Optionally build your docs in additional formats such as PDF
+formats:
+  - pdf
+
+# Optionally set the version of Python and requirements required to build your docs
+python:
+  version: 3.7
+  install:
+    - requirements: docs/requirements.txt
+    - method: pip
+      path: .
+  system_packages: true
```

Comment:
Why set this one to true? I'm afraid this might hide weird bugs later by picking a different version of some dependency

Reply:
Thats true, it was in the copy of the yml file, I'll remove it

Reply:
by removing it it switches to the default behaviour which is false